ice40/utils/ice40_import_bel_timing.py

```diff
@@ -0,0 +1,316 @@
+"""Import timing information from icestorm generated SDF file and updates
+timing in the architecture file.
+"""
+
+import argparse
+from sdf_timing.sdfparse import parse as sdf_parse
+import lxml.etree as ET
+import re
+import logging
+from collections import namedtuple
+
+
+class pin_map(namedtuple('pin_map', 'sdf arch is_clk')):
+    """
+    """
+
+
+"""
+arch_name  - (sdfcelltype, [pin_map ] )
+"""
+
+_arch_to_sdf = {
+    'LUT4':
+        (
+            'LogicCell40', [
+                pin_map('in0', 'in[0]', False),
+                pin_map('in1', 'in[1]', False),
+                pin_map('in2', 'in[2]', False),
+                pin_map('in3', 'in[3]', False),
+                pin_map('lcout', 'out', False)
+            ]
+        ),
+    'SB_LUT4':
+        (
+            'LogicCell40', [
+                pin_map('in0', 'I0', False),
+                pin_map('in1', 'I1', False),
+                pin_map('in2', 'I2', False),
+                pin_map('in3', 'I3', False),
+                pin_map('lcout', 'O', False)
```

Comment:
Any reason not to modify the names to match in either the sdf file or in the sim/xml files?

Reply:
That had crossed my mind, but felt like there are likely cases that we still need this and need to denote if it's a clock.